coap-gateway/service/refreshToken.go

```diff
@@ -79,7 +78,6 @@ func updateClient(client *session, deviceID, owner, accessToken string, validUnt
 		UserID:      owner,
 		AccessToken: accessToken,
 		Expire:      validUntil,
-		JWTClaims:   jwtClaims,
```

Comment:
Wasn't this used in `msgToLogCoapMessage`? 

```go
	if v, err := c.GetAuthorizationContext(); err == nil {
		sub = v.GetJWTClaims().Subject()
	}
```

Is the subject claim never filled so its not necessary to assign here or what's the deal?